.gitignore

```
@@ -2,4 +2,9 @@
 conf.py
 legba.db
 __pycache__
-venv
\ No newline at end of file
+venv
+/data/
```

Was the slash intentional? If not, consistency and removing it seems better.
```suggestion
/data
```